ui/index.d.ts

```diff
@@ -1174,6 +1177,11 @@ export declare namespace DialogResponses {
  */
 export declare function registerUIExtensionVariables(extVars: UIExtensionVariables): void;
 
+/**
+ * Call this to register the experimentation service adapter
+ */
+export declare async function registerExperimentationService(ctx: ExtensionContext, targetPopulation?: TargetPopulation): Promise<void>;
```

Comment:
Idk "register" makes it sound required to me. How about we either update the docs to say "Optional function to register..." or refactor to `createExperimentationService` which just returns the `experimentationService` instead of actually registering it

Reply:
That's a good idea. It also eliminates the need to deal with extension variables.